frontend/app/tests/e2e/pages/defi-page.ts

```diff
@@ -19,9 +19,14 @@ export class DefiPage {
         continue;
       }
 
-      cy.get(`#defi-module-${module}`).scrollIntoView();
-      cy.get(`#defi-module-${module}`).find('button').click();
-      cy.get(`#defi-module-${module}`).should('not.exist');
+      const moduleId = `#defi-module-${module}`;
+      cy.get('body').then($body => {
+        if ($body.find(moduleId).length > 0) {
+          cy.get(moduleId).scrollIntoView();
+          cy.get(moduleId).find('button').click();
+          cy.get(moduleId).should('not.exist');
+        }
+      });
```

Comment:
This change should be reverted I and instead, we should remove the entries for the new modules that are now deactivated by default